src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/cancel.html.twig

```diff
@@ -0,0 +1,25 @@
+{% block toolbar %}
+    {% set icon %}
+        {{ include('@WebProfiler/Icon/symfony.svg') }}
+
+        <span class="sf-toolbar-value sf-toolbar-ajax-request-counter">
+            Loading&hellip;
+        </span>
+    {% endset %}
+
+    {% set text %}
+        <div class="sf-toolbar-info-piece">
+            <b>Loading the web debug toolbar&hellip;</b>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            Attempt #<span id="sfLoadCounter-{{ token }}"></span>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            <b>
+                <button class="cancel-button" type="button" id="sfLoadCancel-{{ token }}" title="Cancel loading">Cancel</button>
```

Comment:
the class should be prefixed with `sf-`, to avoid conflicting with project styles by mistakes (which might apply weird styles on this button). The toolbar does not get rendered in a context where we control all the styles.